### ISSUE TITLE
Bleeds

### ISSUE THREAD

**Issue by ahmetcetin** [diegomura/react-pdf#830]
Is there a possibility to add bleeds (https://www.digitalprinting.co.uk/support/what-is-bleed/) to the produced pdf?

**Comment by geerthoekzema** [comment]
Doesn't it make more sense to 'design' your PDF with bleed (and possible cutmarks)? So if the size is A4, just add 3 mm to each side and give the page, or relevant Views, 3 mm extra padding on each side?

**Comment by digibake** [comment]
I agree with @ahmetcetin, a bleedOffset feature would be absolutely invaluable.
In most Adobe software (like illustrator) the bleeds, slugs and trim marks are features that you set in addition to the page size. It means that the person printing the PDF has the control of whether to use the trim marks or not.
I had a quick scan through the Acrobat DC SDK Documentation and it looks like there's a BleedOffset and UseDocumentBleed. I don't know if that helps?

https://help.adobe.com/en_US/acrobat/acrobat_dc_sdk/2015/HTMLHelp/#t=Acro12_MasterBook%2FPDF_Create_NewNamespaces%2FBleedOffset.htm&rhsearch=bleed&rhhlterm=bleed&rhsyns=%20

A bleed feature would be really helpful for a project that I'm working on. The front of the page has been designing in inDesign and has an 8mm bleed. The back is going to be generated with react-pdf (hopefully) but it's important that the pages are the same size with the same settings otherwise the printing company can't print it. So I'm a bit stuck without that.
